### PR TITLE
Post favourites

### PR DESCRIPTION
**Pull request by Aleksanrda**

### CODE REVIEW

File: models/favourite.models.js [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+        console.log(newFavourite.clothes_id);
+        
+        if (!newFavourite.clothes_id) {
+            throw { 
+                status: 400, 
+                msg: "Bad request!" 
+            };
+        }
+        if(typeof newFavourite.clothes_id !== "number") {
+            return Promise.reject({
```
Comment (Superjim) [inline models/favourite.models.js:27]: does throw not work here?

**Review by Superjim — APPROVED** [review]
looks really good, will be nicer to work on

File: models/favourite.models.js (hunk 1 above)
Comment (Aleksanrda) [reply models/favourite.models.js]: nice catch, fixed it!

Comment (Superjim) [reply models/favourite.models.js]: sorry i didnt mean it was wrong or a problem, just under some circumstances you cant throw an error it has to be promise rejected, i think its only when the throw is nested in a promise though

**Pull request merged**